python/cugraph/cugraph/centrality/katz_centrality.py

```diff
@@ -11,22 +11,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cugraph.centrality import katz_centrality_wrapper
+from pylibcugraph.experimental import (ResourceHandle,
+                                       GraphProperties,
+                                       SGGraph,
+                                       katz_centrality as pylibcugraph_katz
+                                       )
 from cugraph.utilities import (ensure_cugraph_obj_for_nx,
                                df_score_to_dictionary,
                                )
+import cudf
 
 
 def katz_centrality(
-    G, alpha=None, beta=None, max_iter=100, tol=1.0e-6,
+    G, alpha=None, beta=1.0, max_iter=1000, tol=1.0e-6,
```

Comment:
These defaults were changed to reflect networkx's defaults in their katz_centrality implementation, so while it would make the changes breaking, it would also help with networkx compat. 

Another problem that I see is that while the default for alpha was and is None without problems, having beta's default being None would make the default call fail, regardless of an added `TypeError` (cugraph.katz_centrality(G) would fail by default)

I can revert them back to the original defaults if so.